python/BioSimSpace/FreeEnergy/_atm.py

```diff
@@ -0,0 +1,1393 @@
+######################################################################
+# BioSimSpace: Making biomolecular simulation a breeze!
+#
+# Copyright: 2017-2023
+#
+# Authors: Lester Hedges <lester.hedges@gmail.com>, Matthew Burman <matthew@openbiosim.org>
+#
+# BioSimSpace is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# BioSimSpace is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with BioSimSpace. If not, see <http://www.gnu.org/licenses/>.
+######################################################################
+
+# Functionality for creating and viewing systems for Atomic transfer.
+__all__ = ["AToM"]
+
+
+from .. import _is_notebook
+from .._SireWrappers import Molecule as _Molecule
+from .._SireWrappers import System as _System
+from .. import _Utils
+from ..Types import Length as _Length
+from ..Types import Vector as _Vector
+from ..Types import Coordinate as _Coordinate
+from ..Align import matchAtoms as _matchAtoms
+from ..Align import rmsdAlign as _rmsdAlign
+from ..Notebook import View as _View
+from ..Process import OpenMM as OMMprocess
```

Comment:
Can this be private, e.g. `from ..Process import OpenMM as _OpenMM`.

Reply:
Now private (https://github.com/OpenBioSim/biosimspace/blob/feature_AToM/python/BioSimSpace/FreeEnergy/_atm.py#L36)